rvsol/foundry.toml

```diff
@@ -4,5 +4,19 @@ out = 'out'
 libs = ['lib']
 optimizer = true
 optimizer_runs = 999999
+remappings = [
+  '@optimism/packages/contracts-bedrock=lib/optimism/packages/contracts-bedrock',
```

Comment:
We can shorten this remapping to `@optimism/=lib/optimism/packages/contracts-bedrock/`